etc/sample.opennebula.yaml

```diff
@@ -87,8 +87,9 @@ compute:
     shares:
         # the vo name
         ops:
-            # Group name in OpenNebula, default is equal to share name
-            #project: ops
+            #auth:
+            #   # Group name in OpenNebula, default is equal to share name
+            #   project_id: xxxxx
```

Comment:
Here it should be `group`, not `project_id` and it will impact the code in the OpenNebula provider.

Reply:
@gwarf there is this check in `cloud_info_provider/providers/static.py`, common to all providers:
https://github.com/EGI-Foundation/cloud-info-provider/blob/580945c57160fde4dcc5c8deb818fe1dc0ccaa2c/cloud_info_provider/providers/static.py#L146-L147

that will be affected if we change `project_id` to `group`. How do you plan to address this? moving this check to openstack provider?

Reply:
@enolfc what's your feeling on this?

Reply:
These lines actually belong to the OpenNebula provider as this default group = vo is a rOCCI convention. So for sure I would remove them from static.py. For the time being we can even remove the convention and force ONE users to have explicit "group" in the config, fill an issue and fix this later.